kvm.go

```diff
@@ -684,6 +725,44 @@ func createDiskImage(dest string, size int, r io.Reader) error {
 	return f.Close()
 }
 
+func (d *Driver) createDiskVolume(size int, buf *bytes.Buffer) error { 
+	raw := buf.Bytes()
+	sizeBytes := int64(size) << 20
+	pool, err := d.conn.LookupStoragePoolByName(d.StoragePoolName)
+	if err != nil {
+		return err
+	}
+	xml := fmt.Sprintf(newVolumeXML, d.StorageVolumeName, sizeBytes, sizeBytes)
+	volume, err := pool.StorageVolCreateXML(xml,0);
```

Comment:
I believe you'll need a corresponding call to `StorageVolDelete(...)` http://libvirt.org/html/libvirt-libvirt-storage.html#virStorageVolDelete most likely from within `Remove()` - In my testing of this PR it looks like it's leaking volumes. Upon removal of the machine, the storage volume still exists in the pool.